### ISSUE TITLE
levdopa incorrectly split

### ISSUE THREAD

**Issue by cbizon** [TranslatorSRI/Babel#114]
There are 2 identifiers for levdopa in NN:
PUBCHEM.COMPOUND:6047 (A small molecule)
UMLS:C0023579 (A Protein)

Levdopa is not a protein, but it is an amino acid.  I suspect that this is an over-zealous assignment of a class to a UMLS branch that is probably something like "proteins, polypeptides, and amino acids"

**Comment by gaurav** [comment]
Which NN are you looking at? They are merged in 2022dec2-2 on ITRB Prod: https://nodenorm.transltr.io/1.3/get_normalized_nodes?curie=PUBCHEM.COMPOUND%3A6047&conflate=true

**Comment by cbizon** [comment]
Right, sort of -  I hadn't noticed that the UMLS is part of the PUBCHEM clique.  But, the UMLS also has its own:
https://nodenorm.transltr.io/1.3/get_normalized_nodes?curie=UMLS%3AC0023570&conflate=true

That should bring back the pubchem document above, but doesn't

**Comment by gaurav** [comment]
Hmm. Let me check the actual JSON output and let you know what I find.

**Comment by gaurav** [comment]
This is related to https://github.com/TranslatorSRI/Babel/issues/110 in that they both have the same identifier in different cliques, which I didn't think was something the Redis graph supported.

**Comment by gaurav** [comment]
Just an update: [PUBCHEM.COMPOUND:6047 in NodeNorm Dev](https://nodenormalization-sri.renci.org/1.4/get_normalized_nodes?curie=PUBCHEM.COMPOUND%3A6047&conflate=true&drug_chemical_conflate=false&description=false) is now classified as a SmallMolecule rather than a protein/amino acid. I don't see an entry for [UMLS:C0023579 in UMLS at all](https://uts.nlm.nih.gov/uts/umls/searchResults?searchString=C0023579), but PUBCHEM.COMPOUND:6047 is cliqued with [UMLS:C0023570 "levodopa"](https://uts.nlm.nih.gov/uts/umls/concept/C0023570). So I think everything in this ticket is resolved?

**Comment by gaurav** [comment]
Oh wait, no, the problem is actually that UMLS:C0023570 is showing up in both [clique UMLS:C0023570](https://nodenormalization-sri.renci.org/1.4/get_normalized_nodes?curie=UMLS%3AC0023570&conflate=true&drug_chemical_conflate=false&description=false) *and* in [clique PUBCHEM.COMPOUND:6047](https://nodenormalization-sri.renci.org/1.4/get_normalized_nodes?curie=PUBCHEM.COMPOUND%3A6047&conflate=true&drug_chemical_conflate=false&description=false). Weird.

**Comment by cbizon** [comment]
The 9 was probably a typo...  I agree that it's resolved

**Comment by cbizon** [comment]
ah ok
